acceptance/tests/windows/env_windows_installdir_fact.rb

```diff
@@ -5,17 +5,21 @@
 
   confine :to, :platform => 'windows'
 
+  require 'json'
+
   agents.each do |agent|
-    step "test for presence/accurance of fact on #{agent}" do
+    step "test for presence/accuracy of fact on #{agent}" do
       platform = agent[:platform]
       ruby_arch = agent[:ruby_arch] || 'x86' # ruby_arch defaults to x86 if nil
 
       install_dir = platform =~ /-64$/ && ruby_arch == 'x86' ?
-        "C:\\\\Program Files (x86)\\\\Puppet Labs\\\\Puppet" :
-        "C:\\\\Program Files\\\\Puppet Labs\\\\Puppet"
+        "C:\\Program Files (x86)\\Puppet Labs\\Puppet" :
+        "C:\\Program Files\\Puppet Labs\\Puppet"
 
-      on agent, puppet('facts') do
-        assert_match(/"env_windows_installdir": "#{install_dir}"/, stdout, "env_windows_installdir fact did not match expected output")
+      on agent, puppet('facts', '--render-as json') do
+        facts = JSON.parse(stdout)
```

Comment:
Actually, I think I've more frequently seen `result.stdout`, when given:

```ruby
on agent, puppet('facts', '--render-as json') do |result|
```

I guess `stdout` is magically available even without anything passed to the block per - https://github.com/puppetlabs/beaker/blob/ea263617bcb3e4d4a27c65c48bd55c56c8064612/lib/beaker/dsl/helpers/host_helpers.rb#L85-L94

But it also looks like `stdout` is deprecated - https://github.com/puppetlabs/beaker/blob/ea263617bcb3e4d4a27c65c48bd55c56c8064612/lib/beaker/dsl/helpers/host_helpers.rb#L128-L136